jpa/keyset-pagination/boot-data-window-pagination/src/main/java/com/example/keysetpagination/web/controllers/AnimalController.java

```diff
@@ -65,15 +65,17 @@ PagedResult<AnimalResponse> getAllAnimals(
             })
     @PostMapping("/search")
     public Window<AnimalResponse> searchAnimals(
-            @Parameter(description = "Number of items per page (max 100)")
+            @Parameter(description = "Number of items per page (max 100)", in = ParameterIn.QUERY)
```

Comment:
@coderabbitai, write integration test for capturing min and max value for pageSize